frameworks/Nim/nawak/setup.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+export PATH="$IROOT/nim/bin:$IROOT/mongrel2/bin:$PATH"
+
+sed -i 's|host=.* port=5432|host='"${DBHOST}"' port=5432|g' model_postgre.nim
+
+echo "** Compiling app"
+nim c --threads:on -d:release -d:postgre_model -l:-Wl,-rpath,$IROOT/zeromq-4.0.3/lib -o:nawak_postgre app.nim
+
+echo "** Starting mongrel2"
+cd conf
+mkdir -p run logs tmp
+m2sh load -config mongrel2.conf
+m2sh start -name test &
```

Comment:
Logging might be getting hidden because `m2sh` is being run in the background via the ampersand, maybe try the following:
`m2sh start -name test 1>$OUT 2>$ERR &`
which should force stdout to the normal log file and stderr to the normal err file (results/latest/logs/[test]/err.log).